ios/src/platform_ios.mm

```diff
@@ -54,22 +54,33 @@ bool isContinuousRendering() {
 
 NSString* resolvePath(const char* _path) {
 
-    NSString* path = [NSString stringWithUTF8String:_path];
+    NSString* pathString = [NSString stringWithUTF8String:_path];
+
+    NSURL* resourceFolderUrl = [NSURL fileURLWithPath:[[NSBundle mainBundle] resourcePath]
```

Comment:
You can simplify this by calling `[[NSBundle mainbundle] resourceURL]` which produces a file-based URL to the directory.